src/genbenchQC/report/split_html_report.py

```
@@ -64,57 +240,76 @@ def get_train_test_html_template(clusters, filename_train, sequences_train, file
+        html_template = put_data(html_template, "{{icon_data_leakage}}", '<span class="status-icon status-pass">✔</span>')
+        return html_template
+
+    html_template = put_data(html_template, "{{icon_data_leakage}}", '<span class="status-icon status-fail">✘</span>')
+    results_display = results_filt.head(100).copy() # limit to top 100 hits for display in HTML report
+
+    rows = []
+
+    for i, row in results_display.iterrows():
+        try: 
+            alignment_str_color = build_alignment_string(row, color=True)
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)